platforms/core-configuration/configuration-cache/src/main/kotlin/org/gradle/internal/cc/impl/ConfigurationCacheHost.kt

```diff
@@ -16,209 +16,17 @@
 
 package org.gradle.internal.cc.impl
 
-import org.gradle.api.artifacts.component.BuildIdentifier
-import org.gradle.api.internal.BuildDefinition
-import org.gradle.api.internal.GradleInternal
-import org.gradle.api.internal.SettingsInternal
-import org.gradle.api.internal.initialization.ClassLoaderScope
-import org.gradle.api.internal.initialization.ScriptHandlerFactory
-import org.gradle.api.internal.project.ProjectInternal
-import org.gradle.api.internal.project.ProjectStateRegistry
-import org.gradle.internal.cc.base.serialize.service
-import org.gradle.execution.plan.ScheduledWork
-import org.gradle.groovy.scripts.TextResourceScriptSource
-import org.gradle.initialization.ClassLoaderScopeRegistry
-import org.gradle.initialization.DefaultProjectDescriptor
-import org.gradle.initialization.DefaultSettings
-import org.gradle.initialization.SettingsState
-import org.gradle.initialization.layout.BuildLayout
 import org.gradle.internal.Factory
-import org.gradle.internal.build.BuildState
-import org.gradle.internal.build.BuildStateRegistry
-import org.gradle.internal.build.RootBuildState
-import org.gradle.internal.file.PathToFileResolver
-import org.gradle.internal.reflect.Instantiator
-import org.gradle.internal.resource.TextFileResourceLoader
-import org.gradle.internal.service.CloseableServiceRegistry
-import org.gradle.internal.service.ServiceRegistry
-import org.gradle.internal.service.scopes.ServiceRegistryFactory
-import org.gradle.internal.service.scopes.SettingsScopeServices
-import org.gradle.util.Path
+import org.gradle.internal.cc.base.serialize.HostServiceProvider
 import java.io.File
 
+interface ConfigurationCacheHost : HostServiceProvider {
 
-class ConfigurationCacheHost internal constructor(
```

Comment:
ℹ️  The implementation is now called `DefaultConfigurationCache` and the `ConfigurationCacheHost` is now an interface